frontend/src/pages/api/subscribe.ts

```diff
@@ -0,0 +1,73 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
(thought) The [app router](https://github.com/navapbc/template-application-nextjs/pull/255) just landed in the template. The api would be a logical place to introduce the app router as it NextJS [recommends incrementally updating to the app router](https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#migrating-from-pages-to-app) since the app and page routers can be used simultaneously. This could be an opportunity to introduce the app router, depending on your energy to invest in that.

Reply:
Yes, I'd suggest breaking that out into another ticket as this PR has gotten pretty big so far.